src/main/public/partials/adminFiltering.html

```diff
@@ -0,0 +1,102 @@
+<div layout="column" layout-margin>
+    <div class="" admin-filtering layout="row" layout-margin>
+        <div>
+            <md-card>
+                <md-card-content>
+                    <h4>Filters</h4>
+                    <sbx-grid
+                            load-page="loadFilters(startIndex, count)"
+                            page-size="20"
+                            refresh-button="true"
+                            object-actions="filterObjectActions"
+                            sorting="false"
+                            callbacks="callbacks.filtersTable"
+                            object-selected="filterSelected(object)"
+                            empty-message="No filters added">
+                        <sbx-grid-column property="name" title="Name">
+                        </sbx-grid-column>
+                        <sbx-grid-column property="tagFilterType" title="Type">
```

Comment:
We could consider adding an angular filter here which turns values to lowercase (captialized first letter). It's kind of screaming BLACKLIST! WHITELIST! at the user right now... Maybe just adding this (all lowercase) is simplest: https://docs.angularjs.org/api/ng/filter/lowercase